cwl_utils/parser/cwl_v1_0_utils.py

```diff
@@ -0,0 +1,40 @@
+import hashlib
+from typing import cast
+
+from schema_salad.exceptions import ValidationException
+from schema_salad.utils import json_dumps
+
+from cwl_utils.parser.cwl_v1_0 import CommandLineTool, CommandOutputBinding
+
+
+def convert_stdstreams_to_files(clt: CommandLineTool):
+    for out in clt.outputs:
+        if out.type == 'stdout':
+            if out.outputBinding is not None:
+                raise ValidationException(
+                    "Not allowed to specify outputBinding when using stdout shortcut.")
+            if clt.stdout is None:
+                clt.stdout = str(hashlib.sha1(json_dumps(  # nosec
+                    clt.save(), sort_keys=True).encode('utf-8')).hexdigest())
+            out.type = 'File'
+            out.outputBinding = CommandOutputBinding(glob=clt.stdout)
+        elif out.type == 'stderr':
+            if out.outputBinding is not None:
+                raise ValidationException(
+                    "Not allowed to specify outputBinding when using stderr shortcut.")
+            if clt.stderr is None:
+                clt.stderr = str(hashlib.sha1(json_dumps(  # nosec
+                    clt.save(), sort_keys=True).encode('utf-8')).hexdigest())
+            out.type = 'File'
+            out.outputBinding = CommandOutputBinding(glob=clt.stderr)
+    for inp in clt.inputs:
+        if inp.type == 'stdin':
```

Comment:
CWL v1.0 didn't have a `stdin` type shortcut; that was introduced in CWL v1.1 :-)

https://www.commonwl.org/v1.1/CommandLineTool.html#Changelog

Reply:
I didn't know it, but apparently neither [the parser](https://github.com/common-workflow-language/cwl-utils/blob/661564d13eb09efb6e359a2a4dc8c7c6db0bc0ff/cwl_utils/parser/cwl_v1_0.py#L6932) was aware of that XD